project/Dependencies.scala

```diff
@@ -6,23 +6,29 @@ import sbt.Keys._
 object Dependencies {
 
   object Versions {
-    val akka = "2.5.17"
-
-    val scalaTest = "3.0.5"
+    val akka = "2.6.4"
     val scalaJava8Compat = "0.9.0"
```

Comment:
This shouldn't be needed, it's a transitive dep from Akka. Thereby we avoid the trouble with different versions for Scala 2.12 and 2.13.

Reply:
Hmm, I don't know why I added it. I will remove.